app/assets/locales/locale-en.json

```diff
@@ -989,7 +989,7 @@
                 "tooltip_resolution_date": "The resolution date is an informal statement when the asset owner intends to resolve this betting market."
             },
             "price_feed": {
-                "external_feed_price": "Feed price (derived from price feeds and other parameters)",
+                "external_feed_price": "Settlement price (derived from price feeds and other parameters)",
```

Comment:
If we switch the wording I think there are a dozen other places we need to do that, non?

Reply:
I believe no; only this one is confusing, because it isn't feed price, "external_feed_price" is perhaps right but a complex translation.